### DISCUSSION TITLE
Auto updafe extra_module_url suffix?

### DISCUSSION THREAD

**Discussion by pesschap** [elchininet/custom-sidebar#203] [General]
Hi,

I know we need to put the version number ?v1.0.0 at the end of the extra_module_url for the latest version to load.

Is there a way to keep this version number automatically updated to the latest so I don't have to change it every update?

**Comment by elchininet** [comment]
>I know we need to put the version number ?v1.0.0 at the end of the extra_module_url for the latest version to load.

Hi @iamthekings5,

That is not mandatory, I need to clarify that in the README. That is just a recommendation, Home Assistant should load always the last version but its caching system is sometimes stubborn, so if you face caching issues, that would be the first thing to do to see if it fixes it.

>Is there a way to keep this version number automatically updated to the latest so I don't have to change it every update?

This is a `JavaScript` plugin, not an integration, we don't have access to anything in the server. I recommend you to let it as it is and just change it if you face any caching issue.
